static/docs/commands-reference/version.md

```diff
@@ -0,0 +1,40 @@
+# version
+
+This command shows the system/environment information along with the DVC version
+
+## Synopsis
+
+```usage
+    usage: dvc version [-h] [-q | -v]
+```
+
+## Options
+
+* `-h`, `--help` - prints the usage/help message, and exit.
+
+* `-q`, `--quiet` - does not write anything to standard output. Exit with 0 if
+  no problems arise, otherwise 1.
+
+* `-v`, `--verbose` - displays detailed tracing information.
+
+## Details
+
+Running the command [`dvc version`](/doc/commands-reference/version) outputs the following information about the system/environment:
```

Comment:
*code style*: 80 symbols wrap (except certain cases like long URLs, tables, etc). I recommend using VS Code with the [Rewrap](https://marketplace.visualstudio.com/items?itemName=stkb.rewrap) plugin.